MozillaFoundation/foundation.mozilla.org
# Add link for mozfest schedule

# Description

This PR adds a Schedule link to the mozfest website

Link to sample test page:
Related PRs/issues: TP1-1480 / #13081

# To Test

1. Check out branch locally
2. Navigate to mozfest.localhost:8000 and notice there is now a "Schedule" link in the primary nav
3. Verify Schedule appears on mobile as well

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1484)
